src/components/action-popover/action-popover.component.js

```diff
@@ -184,7 +221,7 @@ ActionPopover.propTypes = {
   /** Render a custom menu button to override default ellipsis icon */
   renderButton: PropTypes.func,
   /** Set whether the menu should open above or below the button */
-  placement: PropTypes.oneOf(["bottom", "top"]),
+  placement: PropTypes.oneOf(["top", "bottom"]),
```

Comment:
This was actually introduced for when the popover would be displayed on the last row of a table, and that table was at the bottom of the page. Won't popper automatically cater for this, if so I think we can deprecate this prop.